document-generation/eval-plan-document.test.ts

```diff
@@ -0,0 +1,38 @@
+import { EvaluationPlan } from "../models/document-generation";
+import { sampleEvalPlanRequest } from "./utils/sampleTestData";
+import * as fs from "fs";
+import * as path from "path";
+import { ApiBase64SuccessResponse, ErrorStatusCode, OtherErrorResponse } from "../utils/response";
+import { generateEvalPlanDocument } from "./eval-plan-document";
+
+describe("Generate an Evaluation Plan document - success path", () => {
+  const sampleEPRequest = sampleEvalPlanRequest.templatePayload as EvaluationPlan;
+  const sampleEPCustomSpecsRequest = sampleEvalPlanRequest.templatePayload as EvaluationPlan;
+
+  it.each([sampleEPRequest, sampleEPCustomSpecsRequest])("should return an ApiBase64Response", async (epPayload) => {
+    // GIVEN
+    const templateBuffer = fs.readFileSync(path.resolve(__dirname, "templates/ifp-template.docx"));
```

Comment:
Not sure if it was intentional or not, but I might want to update the template path.

Reply:
This was unintentional, fixed in https://github.com/dod-ccpo/atat-web-api/pull/1381/commits/619ea2ac7b0ace6681d64e26cc2d39871dd631d9